src/aics-image-viewer/components/App/index.tsx

```diff
@@ -534,37 +540,48 @@ const App: React.FC<AppProps> = (props) => {
 
   const resetCamera = useCallback((): void => view3d.resetCamera(), []);
 
+  const setIndicatorPositions = (panelOpen: boolean, hasTime: boolean): void => {
```

Comment:
since this is just a plain function, would it be slightly more efficient potentially to lift it up out of this component and pass in view3d?

Reply:
I figure every time the react component's "function" is invoked, you are paying the cost of creating this new inner function every time